device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsSendResult.java

```diff
@@ -14,17 +14,16 @@
     /**
      * Create a return value object containing the delivery status and the delivery hash
      *
-     * @param deliverySuccessful the delivery state
      */
-    AmqpsSendResult(boolean deliverySuccessful)
+    AmqpsSendResult()
     {
-        this.deliverySuccessful = deliverySuccessful;
+        this.deliverySuccessful = false;
         this.deliveryTag = failedDeliveryTag;
```

Comment:
the delivery tag here is always updated to the "failed delivery tag", so this method is expected to be always called from the "failed" scenario.

Reply:
This one is public, is it safe to remove here and below?

Reply:
it is not public, it is package-private.